src/views/Votes.vue

```diff
@@ -83,162 +70,128 @@
   </div>
 </template>
 
-<script>
-import AppToolbarCentered from '@/components/AppToolbarCentered.vue'
-import Pagination from '@/components/Pagination.vue'
+<script setup lang="ts">
+import PaginationComponent from '@/components/Pagination.vue'
 import DelegatesTable from '@/components/DelegatesTable/DelegatesTable.vue'
-import { computed, onMounted, ref, reactive, defineComponent, watch } from 'vue'
+import { computed, onMounted, ref, reactive, watch } from 'vue'
 import { useStore } from 'vuex'
 import { useI18n } from 'vue-i18n'
 import { mdiMagnify } from '@mdi/js'
-
+import { DelegateDto } from '@/lib/schema/client'
 
 const VOTE_REQUEST_LIMIT = 30
 
-export default defineComponent({
-  components: {
-    DelegatesTable,
-    AppToolbarCentered,
-    Pagination
-  },
-  setup() {
-    const store = useStore()
-    const { t } = useI18n()
-    const search = ref('')
-    const pagination = reactive({
-      rowsPerPage: 50,
-      sortBy: 'rank',
-      page: 1
-    })
-    const expanded = ref([])
-    const waitingForConfirmation = ref(false)
-    const dialog = ref(false)
-    const className = 'delegates-view'
+const store = useStore()
+const { t } = useI18n()
+const search = ref('')
+const pagination = reactive({
+  rowsPerPage: 50,
+  sortBy: 'rank',
+  page: 1
+})
+const waitingForConfirmation = ref(false)
+const dialog = ref(false)
+const className = 'delegates-view'
 
-    const delegates = computed(() => {
-      const delegates = store.state.delegates.delegates || {}
+interface extendedDto extends DelegateDto {
+  upvoted?: unknown
+  voted?: unknown
+  downvoted?: unknown
+  _voted?: unknown
+}
```

Comment:
Potential conflicts with this PR https://github.com/Adamant-im/adamant-im/pull/746

https://github.com/Adamant-im/adamant-im/blob/6fa3065c5ec13ead9c5bb2b255d6c9ae19fc8b53/src/views/Votes.vue#L97-L105

Reply:
The interface must start with uppercase letter